File: 2154.将找到的值乘以-2.java
/*
 * @lc app=leetcode.cn id=2154 lang=java
 *
 * [2154] 将找到的值乘以 2
 */

// @lc code=start
class Solution {
    public int findFinalValue(int[] nums, int original) {
        while (true) {
            boolean found = false;
            for (int num : nums) {
                if (num == original) {
                    original *= 2;
                    found = true;
                    break;
                }
            }
            if (!found) {
                break;
            }
        }
        return original;
    }
}
// @lc code=end

File: 2154.将找到的值乘以-2.js
/*
 * @lc app=leetcode.cn id=2154 lang=javascript
 *
 * [2154] 将找到的值乘以 2
 */

// @lc code=start
/**
 * @param {number[]} nums
 * @param {number} original
 * @return {number}
 */
var findFinalValue = function(nums, original) {
    while (nums.includes(original)) {
        original *= 2;
    }
    return original;
};
// @lc code=end

